### PR TITLE
BZ2059762: Adds known issue to 4.9.23 release notes

### PR DESCRIPTION
**Pull request by opayne1**
[BZ link](https://bugzilla.redhat.com/show_bug.cgi?id=2059762)

- Adds known issue to 4.9.23 release. Applies to `enterprise-4.9` only
- [Preview link](https://deploy-preview-42784--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes#ocp-4-9-23-known-issues)

### CODE REVIEW

**Comment by opayne1** [comment]
@wking @stevsmit @TanyaShearonRH @sjstout @sferich888 PTAL for change management. Thank you!

**Review by wking — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -2564,6 +2564,11 @@ Space precluded documenting all of the container images for this release in the
 
 link:https://access.redhat.com/solutions/6762051[{product-title} 4.9.23 container image list]
 
+[id="ocp-4-9-23-known-issues"]
+==== Known Issues
+
+{product-title} release 4.9.23 references images whose manifests use the `application/vnd.oci.image.manifest.v1+json` media type. This might cause problems when mirroring to image registries that do not support OCI media types. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2059762[*BZ#2059762*])
```
Comment (sferich888) [inline release_notes/ocp-4-9-release-notes.adoc:2570]: Do we have a stated workaround for this issue?

File: release_notes/ocp-4-9-release-notes.adoc [hunk 2]
```diff
@@ -735,9 +735,9 @@ When configuring the `docker/config.json` file to allow pods to pull images from
 Node-related metrics and alerts have been enhanced to give you an earlier indication of when the stability of a node is compromised.
 
 [id="ocp-4-9-poison-pill-operator"]
-==== Enhanced remediation with the Poison Pill Operator 
+==== Enhanced remediation with the Poison Pill Operator
 
-You can use the Poison Pill Operator regardless of the cluster installation type, such as Installer Provisioned Infrastructure (IPI) or User Provisioned Infrastructure (UPI). 
+You can use the Poison Pill Operator regardless of the cluster installation type, such as Installer Provisioned Infrastructure (IPI) or User Provisioned Infrastructure (UPI).
```
Comment (sferich888) [inline release_notes/ocp-4-9-release-notes.adoc:740]: Is this statement needed? Doesn't this apply to most/all operators? 

cc: @davemulford

Comment (davemulford) [reply release_notes/ocp-4-9-release-notes.adoc:740]: +1 Eric. I don't think this sentence is needed.

**Pull request merged**